modules/fedimint-mint-client/src/client_db.rs

```diff
@@ -98,3 +99,14 @@ impl_db_lookup!(
     key = CancelledOOBSpendKey,
     query_prefix = CancelledOOBSpendKeyPrefix,
 );
+
+pub async fn migrate_to_v1(
+    dbtx: &mut DatabaseTransaction<'_>,
+) -> anyhow::Result<Option<(Vec<(Vec<u8>, OperationId)>, Vec<(Vec<u8>, OperationId)>)>> {
+    // between v0 and v1, we changed the format of `MintRecoveryState`, and instead
+    // of migrating it, we can just delete it, so the recovery will just start
+    // again, ignoring any existing state from before the migration
+    dbtx.remove_entry(&RecoveryStateKey).await;
```

Comment:
Seems safe to me since I can't find any code that derives the fact that we are recovering from its presence.

Reply:
It was all designed to support discarding the state.